2-0-data-structures-and-algorithms/src/test/java/com/bobocode/stack/StackTest.java

```diff
@@ -0,0 +1,75 @@
+package com.bobocode.stack;
+
+import com.bobocode.stack.exception.EmptyStackException;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.*;
+
+class StackTest {
```

Comment:
Let's add a java doc to say that tests depend on all methods, so you need to implement everything before running the tests.